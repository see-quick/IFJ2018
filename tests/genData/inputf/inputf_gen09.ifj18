# inputf test

a = inputf
a = 5.21 + a 
