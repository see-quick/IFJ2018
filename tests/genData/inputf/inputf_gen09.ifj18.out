0x1.4d70a3d70a3d7p+2
